packages/web-components/src/components/card-group/card-group.ts

```diff
@@ -149,8 +150,14 @@ class DDSCardGroup extends StableSelectorMixin(LitElement) {
       if (!this.pictograms) {
         this._setSameHeight();
       }
-      if (this.gridMode !== GRID_MODE.NARROW) {
+
+      if (gridMode !== GRID_MODE.NARROW) {
         this._fillLastRowWithEmptyCards(columns);
+      } else {
+        this._removeEmptyCards();
+      }
```

Comment:
Previously, you could get leftover empty cards when switching from a non-narrow grid mode to the narrow one. This didn't cause any issues (yet!), but there was an easy opportunity to reuse some existing code to clean up after ourselves.